BrainPortal/app/models/cluster_task.rb

```diff
@@ -2050,7 +2050,14 @@ def docker_commands(command_script)
   def load_docker_image_cmd #:nodoc:
     dockerhub_image_name = self.tool_config.containerhub_image_name
     docker_container_index = self.tool_config.container_index_location
-    docker_container_index += '/' if docker_container_index.present? && !docker_container_index.end_with?('/') # Add trailing / if not present and field not empty
+    if docker_container_index.present?
```

Comment:
Well, that will work, but it would have been simpler to change line 2052 to

```ruby
docker_container_index = self.tool_config.container_index_location.presence || ""
```